packages/loaders/src/styled/StyledInline.ts

```diff
@@ -5,11 +5,24 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled, { DefaultTheme, ThemeProps, keyframes } from 'styled-components';
-import { retrieveComponentStyles } from '@zendeskgarden/react-theming';
+import styled, { DefaultTheme, ThemeProps, css, keyframes } from 'styled-components';
+import { getColor, retrieveComponentStyles } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'loaders.inline';
 
+interface IStyledInlineProps {
+  size: number;
+  color: string;
+}
+
+const colorStyles = ({ theme, color }: IStyledInlineProps & ThemeProps<DefaultTheme>) => {
+  const options = color.includes('.') ? { variable: color, theme } : { hue: color, theme };
```

Comment:
To double-check, we do not support rgba colors (`rgba(123,43,56, 0.5)`. Correct?